docs/kafka.md

```diff
@@ -0,0 +1,145 @@
+# Kafka
+
+Tranquility Kafka is an application which simplifies the ingestion of data from Kafka. It is scalable and highly
+available through the use of Kafka partitions and consumer groups, and can be configured to push data from multiple
+Kafka topics into multiple Druid dataSources.
+
+## Setup
+
+### Getting Tranquility Kafka
+
+Tranquility Kafka currently must be built from source. To build, run `sbt universal:packageZipTarball`. The build output
+will be contained in the archive `/kafka/target/universal/tranquility-kafka-x.x.x.tgz`. Unpack this tarball to the
+directory of choice by running `tar -xzf tranquility-kafka-x-x-x.tgz`.
+
+### Configuration
+
+Tranquility Kafka uses a YAML file for configuration. You can see an example in `conf/tranquility-kafka.yaml.example` of the
+tarball distribution. You can start off your installation by copying this example file to `conf/tranquility-kafka.yaml`. The
+YAML file has two sections:
+
+1. `dataSources` - per dataSource configuration.
+2. `properties` - general properties that apply to all dataSources. See "Configuration reference" for details.
+
+The dataSources key should contain a mapping of dataSource name to configuration. Each dataSource configuration
+has two sections:
+
+1. `spec` - a YAML representation of a [Druid ingestion spec](http://druid.io/docs/latest/ingestion/index.html). Note
+that the `ioConfig` is expected to be of type `realtime` but otherwise empty (null firehose, null plumber). This is
+because Tranquility supplies its own firehose and plumber.
+2. `properties` - per dataSource properties. See "Configuration reference" for details.
+
+### Running
+
+If you've saved your configuration into `conf/tranquility-kafka.yaml`, run the application with:
+
+```bash
+bin/tranquility-kafka -configFile conf/tranquility-kafka.yaml
+```
+
+## Configuration reference
+
+With the exception of the global properties (which must be provided at the top level), most properties can be provided
+at the top level or per-dataSource. If the same property is provided at both levels, the per-dataSource version will
+take precedence. This provides a way to set default properties that apply to all dataSources, while still allowing
+customization for certain dataSources.
+
+### Global properties
+
+|Property|Description|Default|
```

Comment:
Anything that starts in `kafka.` is passed to the consumer, right? that should be mentioned here too.